src/components/HorizontalStackedBarChart/HorizontalStackedBarChart.module.scss

```
@@ -4,7 +4,7 @@
   font-size: 21px;
   font-weight: 700;
 
-  color: #34495E;
+  color: var(--datacite-color-primary, #34495E);
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)